README.md

```diff
@@ -38,6 +38,10 @@ Then with a local `DATABASE_URL` var exported like this `export DATABASE_URL=pos
 
 If you need to install `sea-orm-cli` run `cargo install sea-orm-cli`.
 
+Note: The current SeaORM types were generated using version 0.9.3 so unless you want to upgrade you can install using `cargo install sea-orm-cli --version 0.9.3`.
+
+Also note: The migration `m20230224_093722_performance_improvements` needs to be commented out of the migration lib.rs in order for the Sea ORM `Relations` to generate correctly.
```

Comment:
According to @austbot we need this performance improvement, but the problem is the autogeneration relies on the foreign key constraints to work.  So I think I'm with you that this could be removed, but I didn't want to do it as part of this PR.  I think it would be better to do it as a follow on PR just focused on the effects of those changes.

Reply:
Ah gotcha, this rings a bell from when we added a migration ourselves and it ruined all of our generated models.

Reply:
for what its worth, we don't need the relations to exist on the models to power those queries. it's just a nice to have. we can just restructure the SeaORM query and it will work fine.

Reply:
Ok that sounds good too.  I haven't looked into that yet.  Maybe I can do it as a follow on PR unless its really easy to just do it now.